opennebula/shared_schemas.go

```diff
@@ -618,6 +618,18 @@ func flattenTemplate(d *schema.ResourceData, vmTemplate *vm.Template) error {
 	port, _ := vmTemplate.GetIOGraphic(vmk.Port)
 	t, _ := vmTemplate.GetIOGraphic(vmk.GraphicType)
 	keymap, _ := vmTemplate.GetIOGraphic(vmk.Keymap)
+	// Features
+	featuresMap := make([]map[string]interface{}, 0, 1)
+	pae, _ := vmTemplate.GetFeature(vmk.PAE)
+	acpi, _ := vmTemplate.GetFeature(vmk.ACPI)
+	apic, _ := vmTemplate.GetFeature(vmk.APIC)
+	localtime, _ := vmTemplate.GetFeature(vmk.LocalTime)
+	// not defined in vendor/github.com/OpenNebula/one/src/oca/go/src/goca/schemas/vm/keys/template.go
+	//hyperv, _ := vmTemplate.GetFeature(vmk.Hyperv)
```

Comment:
for all of these commands, did you try something like:
`hyperv, _ := vmTemplate.GetFeature("HYPERV")`

Reply:
> for all of these comments, did you try something like: `hyperv, _ := vmTemplate.GetFeature("HYPERV")` vmk.XXX are only some helpers for peoples that don't necessarily know the keys, and to help with a bit the developer via some IDE autocompletion. By the way, feel free to submit a PR to OpenNebula to add more keys

Added them without using vmk for now and will consider a pull request to OpenNebula in the future.